email.py

```diff
@@ -0,0 +1,12 @@
+import re
+#email validation
+a = 1
+while a==1:
+    txt = input("enter mail id: ")
+    pattern = "[A-Z]+[a-z]+\d+[@!#$%^&*_]+[@][a-z]{5,7}[.][a-z]{2,3}"
+    x = re.search(pattern,txt)
+    if re.search(pattern,txt):
+        print("valid email-id")
+        print(x.string)
+    else:
+        print("invalid email-id")
```

Comment:
From line no 5 to 12 should go inside a function and Return True or False from the function.